SECP-service/src/main/resources/assets/app/views/login/login.html

```diff
@@ -0,0 +1,26 @@
+<div class="container-fluid">
+
+    <div class="form-group">
+        <form autocomplete="on">
+            <h1 id="formHeader">Log in</h1>
+
+            <div class="group">
+                <input type="email" required="required" onkeyup="this.setAttribute('value', this.value);" value="" />
```

Comment:
I don't think you need the onkeyup. Look at "ng-model" for angular. I think it automatically set the value as it gets the input.

Reply:
I can't seem to find a way to use ng-model in a .css file which is needed because i'm setting the css for the labels based on if the inputted value is valid or not